ec2/spark_ec2.py

```diff
@@ -872,9 +890,16 @@ def deploy_files(conn, root_dir, opts, master_nodes, slave_nodes, modules):
     if "." in opts.spark_version:
         # Pre-built Spark deploy
         spark_v = get_validate_spark_version(opts.spark_version, opts.spark_git_repo)
+        tachyon_v = get_tachyon_version(spark_v)
     else:
         # Spark-only custom deploy
         spark_v = "%s|%s" % (opts.spark_git_repo, opts.spark_version)
+        if opts.tachyon_version is None:
```

Comment:
In former commit, I do set a default value, but I think if someone uses git hash and doesn't specify tachyon version, and the default tachyon version doesn't match spark's, then it's hard for he/she to know the error unless he/she has carefully read the document or the command line help or read the src, so I removed it.